styles/style.css

```diff
@@ -0,0 +1,111 @@
+body {
+    background-color: white;
+}
+
+*,
+*:before,
+*:after {
+    margin: 0;
+    padding: 0;
+}
+
+.site-info:after {
+    content: "";
+    display: table;
+    clear: both;
+}
+
+.cf:before,
+.cf:after {
+    content: "";
+    display: table;
+}
+
+.cf:after {
+    clear: both;
+}
+
+.container {
+    width: 960px;
+    margin: 0 auto;
+    background: white;
+}
+
+
+.main-nav {
+    height: 60px;
+    width: 60%;
+    background: yellowgreen;
+    position: relative;
+    float: right;
```

Comment:
You are confusing multiple types of positioning here; the `.logo` is using a `position: absolute` and therefore the parent `.main-header` requires a `position: relative`, which you have.

However, since your `.main-nav` lives inside of the `.main-header` too, then it you should use `position: absolute` here as well (see below).

```css
.main-nav {
  height: 60px;
  width: 60%;
  background: yellowgreen;
  position: absolute;
  top: 0;
  right: 0;
}
```